src/util.py

```diff
@@ -260,12 +265,115 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def set_vector_density(data, lat_density=1, lon_density=1, minDistance=0, otherVars=[]):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        data (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot latitude/longitude data.
```

Comment:
May need to put here the information that latitude/longitude coords of "data" should be named "lat" and "lon", right?